src/main/java/picard/sam/CompareSAMs.java

```diff
@@ -118,6 +118,9 @@ protected int doWork() {
             if (OUTPUT != null) {
                 comparison.writeReport(OUTPUT, getDefaultHeaders());
             }
+            if (samComparisonArgumentCollection.OUTPUT_MQ_CONCORDANCE != null) {
+                comparison.writeMQConcordanceReport(samComparisonArgumentCollection.OUTPUT_MQ_CONCORDANCE, getDefaultHeaders());
```

Comment:
Might want to add a:
        IOUtil.assertFileIsWritable(samComparisonArgumentCollection.OUTPUT_MQ_CONCORDANCE);
before the call to generate the concordance report (in order to fail fast if there's a problem writing the file).